test/main.js

```diff
@@ -30,11 +30,9 @@ describe("gulp-jsoncombine", function () {
 			contents: fs.readFileSync("test/fixtures/hello.txt")
```

Comment:
I couldn't get the tests to pass.  So I made a few changes.  Let me know if they're wrong!